2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md

```diff
@@ -0,0 +1,31 @@
+# Title
+
+Dealing with UEFI Secure Boot support using Yocto Project
+
+# Abstract
+
+UEFI Secure Boot, a standard within the UEFI framework, enhances embedded
+platform security by verifying trusted keys for loaded images in the UEFI BIOS.
```

Comment:
@TomaszAIR, that sounds very weird. There is no key verification but signature verification. There can also be other verifications depending on the configuration. Please at least skim through: https://gitlab.com/opensecuritytraining/arch4221_uefi_secure_boot_slides_and_subtitles/-/merge_requests/5/diffs

Reply:
@pietrushnic fixed https://github.com/3mdeb/conferences/pull/9/commits/7b3b6e2c2fdf6915f719670e1cbbd6578c3b4685 

I got `404 Not found` for the provided link.

Reply:
@TomaszAIR Sorry, I forgot that slides are private. Please check internal repos for the presentation.